standard/toolchain-android.cmake

```diff
@@ -1,8 +1,8 @@
 set(CMAKE_SYSTEM_NAME Android)
 set(CMAKE_SYSTEM_VERSION 21)
-find_path(CMAKE_ANDROID_STANDALONE_TOOLCHAIN sysroot/usr/lib/libandroid.so PATHS
-  /opt/android-standalone-toolchain
-)
+set(BUILD_ARM 1)
+set(BUILD_ANDROID 1)
```

Comment:
These two variables are Leap-isms that I don't think we necessarily want in the public repos (and we currently don't use them).

Reply:
Good catch!!